src/System.Private.CoreLib/shared/System/MissingFieldException.cs

```diff
@@ -2,21 +2,13 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-/*=============================================================================
-**
-**
-** Purpose: The exception class for class loading failures.
-**
-=============================================================================*/
-
-
-using System;
 using System.Runtime.Serialization;
-using System.Runtime.CompilerServices;
-using System.Globalization;
 
 namespace System
 {
+    /// <summary>
+    /// The exception class for class loading failures.
```

Comment:
This description is not correct. You can rather delete it. The exception name is self-explanatory.